.github/workflows/local-pr.yaml

```diff
@@ -0,0 +1,171 @@
+name: "Local PR"
+run-name: "Local PR requested for #${{ github.event.issue.number }} by @${{ github.actor }}"
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  pr_commented:
+    name: Identify comment source
+    if: ${{ github.event.issue.pull_request }} || ${{ github.event.issue.state == 'open' }}
```

Comment:
Is `github.event.issue.state == 'open'` the same as "source = PR"?
Or should this be `&&` instead of `||`?

Reply:
This should be `&&` instead.